packages/snaps-controllers/jest.config.js

```diff
@@ -5,34 +5,14 @@ const baseConfig = require('../../jest.config.base');
 module.exports = deepmerge(baseConfig, {
   coverageThreshold: {
     global: {
-      branches: 88.2,
-      functions: 94.67,
-      lines: 96.26,
-      statements: 96.18,
+      branches: 86.95,
+      functions: 93.08,
```

Comment:
I guess one question is, are we fine with not being able to track code coverage for a bunch of important pieces of the architecture 🤔

Reply:
`webdriverio` coverage does not work under lockdown. It causes the tests to fail, so we can't really collect coverage.

Reply:
Any ideas how we can still enforce/check testing of important features that run under lockdown?

Reply:
All tests that require lockdown can run in `webdriverio`. All tests testing important features, such as hardening, are running in `webdriverio`.

Reply:
Yeah, I'm just saying that without coverage it gets more difficult to check whether important features are tested by `webdriverio`

Reply:
And I'm asking an open-ended question regarding how we feel about that

Reply:
It will require some more careful code review for stuff touching important features, maybe. I personally don't think it's a big issue, and the benefits of `webdriverio` outweigh the drawbacks.